lib/collection/variable-scope.js

```diff
@@ -141,7 +141,7 @@ _.assign(VariableScope.prototype, /** @lends VariableScope.prototype */ {
      */
     toObject: function (excludeDisabled, caseSensitive) {
         // if the scope has no layers, we simply export the contents of primary store
-        if (!this._layers) {
+        if (_.isEmpty(this._layers)) {
```

Comment:
Is `!this._layers` not sufficient for any cases?

`_.isEmpty()` is 100% slower than `!variable`
<img width="981" alt="screen shot 2017-08-10 at 12 49 19 pm" src="https://user-images.githubusercontent.com/8401064/29158916-644c3df4-7dca-11e7-9454-743bd7a84c78.png">